test/unit/style-spec/expression.test.js

```diff
@@ -1,5 +1,48 @@
 import { test } from 'mapbox-gl-js-test';
 import { createPropertyExpression } from '../../../src/style-spec/expression';
+import definitions from '../../../src/style-spec/expression/definitions';
+import v8 from '../../../src/style-spec/reference/v8';
+import {expressions as definitionMetadata} from '../../../docs/components/expression-metadata';
+
+test('expression documentation', (t) => {
+    const equals = (arr1, arr2) => {
+        if (arr1.length !== arr2.length) {
+            return false;
+        }
+
+        arr1.sort();
+        arr2.sort();
+        let i = arr1.length;
+
+        while (i--) {
+            if (arr1[i] !== arr2[i]) {
+                return false;
+            }
+        }
+
+        return true;
+    };
+
+    // filter out interal "error" and "filter-*" expressions from definition list
+    const filterExpressionRegex = /filter-/;
+    const definitionList = Object.keys(definitions).filter((expression) => {
+        return expression !== 'error' && !filterExpressionRegex.exec(expression);
+    });
+
+    test('v8.json includes all definitions from style-spec', (t) => {
```

Comment:
I think this (and the one below) should be `t.test(...)` if we want each of these to be a "subtest".  It's also probably fine to just drop the subtest wrapper and do both of these inline.